modules/rte_mis_lottery/rte_mis_lottery.install

```diff
@@ -59,17 +61,55 @@ function rte_mis_lottery_schema() {
  * Impelements hook_install().
  */
 function rte_mis_lottery_install() {
-  $oauth_keys_folder = '../oauth_keys';
-  // Check if the folder already exists.
-  if (!file_exists($oauth_keys_folder)) {
-    // Create the folder with the desired permissions.
-    mkdir($oauth_keys_folder, 0755, TRUE);
-  }
+  // Define the base schema for the table.
+  $schema = [
```

Comment:
This information is not enough, schools can have seats based on entry_class,

Reply:
Now, Schools can have multiple entries with different entry_class. so school_id cannot be the primary key.